public/components/visualize/wz-visualize.js

```diff
@@ -159,6 +159,11 @@ export const WzVisualize = compose(
     };
     reloadToast = () => {
       const toastLifeTimeMs = 300000;
+      const [mayor, minor] = localStorage.getItem('appInfo') ?
+        JSON.parse(localStorage.getItem('appInfo'))['app-version'].split('.')
+        : ['current'];
+      const wazuhVersion = minor ? `${mayor}.${minor}` : mayor;
+      const urlTroubleShootingDocs = `https://documentation.wazuh.com/${wazuhVersion}/user-manual/kibana-app/troubleshooting.html#no-template-found-for-the-selected-index-pattern`;
```

Comment:
**nitpick**: We could use the value defined in the `package.json` file in the `version` property https://github.com/wazuh/wazuh-kibana-app/blob/4.3-7.10/package.json#L3.

Reply:
Much cleaner :+1: